src/mlpack/methods/dbscan/dbscan.hpp

```diff
@@ -0,0 +1,126 @@
+/**
+ * @file dbscan.hpp
+ * @author Ryan Curtin
+ *
+ * An implementation of the DBSCAN clustering method, which is flexible enough
+ * to support other algorithms for finding nearest neighbors.
+ */
+#ifndef __MLPACK_METHODS_DBSCAN_DBSCAN_HPP
```

Comment:
We should remove the underscore here, because leading or trailing underscores are deprecated? Here is the PR https://github.com/mlpack/mlpack/pull/537